libcaf_net/caf/net/web_socket/framing.cpp

```diff
@@ -10,6 +10,45 @@
 
 namespace caf::net::web_socket {
 
+// -- static utility functions -------------------------------------------------
+
+error framing::validate_closing_payload(const_byte_span payload) {
+  if (payload.empty())
+    return error{};
+  if (payload.size() == 1)
```

Comment:
The status below says "must have two bytes", so shouldn't this read the following?

```suggestion
  if (payload.size() != 2)
```

Reply:
Sadly, no. Empty payload is allowed, meaning no status code at all, 2 bytes means only the status code is present, and everything after the second byte is the closing message.
I'll change the error message though.